images/plbase/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM amazonlinux:2023
 
+ENV PATH="/.venv/bin:$PATH"
```

Comment:
This ensures you aren't accidentally using a system python in your terminal with `python`. Maybe there is a different fix I can do here instead?